backends/p4tools/modules/testgen/lib/exceptions.h

```diff
@@ -1,29 +1,31 @@
 #ifndef BACKENDS_P4TOOLS_MODULES_TESTGEN_LIB_EXCEPTIONS_H_
 #define BACKENDS_P4TOOLS_MODULES_TESTGEN_LIB_EXCEPTIONS_H_
 
+#include "absl/strings/str_cat.h"
 #include "lib/exceptions.h"
 
 namespace P4Tools::P4Testgen {
 
 /// This class indicates a feature that is not implemented in P4Testgen.
 /// Paths with this unimplemented feature should be skipped
+// FIXME: Can we use CompilerUnimplemented instead?
```

Comment:
Iiirc we catch this specific exception in the test-case generator and bubble up other exception. This is why we distinguish the exceptions here.